src/themes/creator.js

```diff
@@ -9,696 +9,45 @@ LICENSE file in the root directory of this source tree.
-      scale100: '4px',
-      scale200: '6px',
-      scale300: '8px',
-      scale400: '10px',
-      scale500: '12px',
-      scale550: '14px',
-      scale600: '16px',
-      scale650: '18px',
-      scale700: '20px',
-      scale750: '22px',
-      scale800: '24px',
-      scale900: '32px',
-      scale1000: '40px',
-      scale1200: '48px',
-      scale1400: '56px',
-      scale1600: '64px',
-      scale2400: '96px',
-      scale3200: '128px',
-      scale4800: '192px',
-    },
-    lighting: {
-      shadow400: '0 1px 4px hsla(0, 0%, 0%, 0.16)',
-      shadow500: '0 2px 8px hsla(0, 0%, 0%, 0.16)',
-      shadow600: '0 4px 16px hsla(0, 0%, 0%, 0.16)',
-      shadow700: '0 8px 24px hsla(0, 0%, 0%, 0.16)',
-      overlay0: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0)',
-      overlay100: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0.04)',
-      overlay200: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0.08)',
-      overlay300: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0.12)',
-      overlay400: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0.16)',
-      overlay500: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0.2)',
-      overlay600: 'inset 0 0 0 1000px hsla(0, 0%, 0%, 0.24)',
-    },
-    borders: {
-      border100: {
-        borderColor: 'hsla(0, 0%, 0%, 0.04)',
-        borderStyle: 'solid',
-        borderWidth: '1px',
-      },
-      border200: {
-        borderColor: 'hsla(0, 0%, 0%, 0.08)',
-        borderStyle: 'solid',
-        borderWidth: '1px',
-      },
-      border300: {
-        borderColor: 'hsla(0, 0%, 0%, 0.12)',
-        borderStyle: 'solid',
-        borderWidth: '1px',
-      },
-      border400: {
-        borderColor: 'hsla(0, 0%, 0%, 0.16)',
-        borderStyle: 'solid',
-        borderWidth: '1px',
-      },
-      border500: {
-        borderColor: 'hsla(0, 0%, 0%, 0.2)',
-        borderStyle: 'solid',
-        borderWidth: '1px',
-      },
-      border600: {
-        borderColor: 'hsla(0, 0%, 0%, 0.24)',
-        borderStyle: 'solid',
-        borderWidth: '1px',
-      },
-      radius100: '2px',
-      radius200: '4px',
-      radius300: '8px',
-      radius400: '12px',
-      /** Checkbox, Datepicker (Range), Progress Bar, Slider, Tag */
-      useRoundedCorners: true,
-      /** Button, ButtonGroup */
-      buttonBorderRadius: '0px',
-      /** Input, Select, Textarea */
-      inputBorderRadius: '0px',
-      /** Popover, Menu, Tooltip */
-      popoverBorderRadius: '0px',
-      /** Card, Datepicker, Modal, Toast, Notification */
-      surfaceBorderRadius: '0px',
-    },
-    animation: {
-      timing100: '0.25s',
-      timing400: '0.4s',
-      timing700: '0.6s',
-      timing1000: '1s',
-      easeOutCurve: 'cubic-bezier(.2, .8, .4, 1)',
-      easeInCurve: 'cubic-bezier(.8, .2, .6, 1)',
-      easeInOutCurve: 'cubic-bezier(0.4, 0, 0.2, 1)',
+      ...colorTokensWithOverrides,
+      ...getColorComponentTokens(colorTokensWithOverrides),
+      ...getDeprecatedSemanticColors(colorTokensWithOverrides),
     },
+    typography: primaryFontFamily
```

Comment:
It took me a bit to figure out what was happening in some of these functions, and I've been working with the themes for a bit now. It might be difficult for users to understand how to theme Base if they reference this. I wonder if there is a way to make these more readable.